README.md

```diff
@@ -3,6 +3,7 @@
 A Python-3 (CPython >= 3.5.0) Interpreter written in Rust :snake: :scream: :metal:.
 
 [![Build Status](https://travis-ci.org/RustPython/RustPython.svg?branch=master)](https://travis-ci.org/RustPython/RustPython)
+[![codecov](https://codecov.io/gh/RustPython/RustPython/branch/master/graph/badge.svg)](https://codecov.io/gh/mozilla/RustPython/RustPython)
```

Comment:
`mozilla/RustPython/RustPython`?
Seems like the link is dead.

Reply:
my bad, bad copy/paste...

Reply:
I just made a PR to fix that:
https://github.com/RustPython/RustPython/pull/491